### DISCUSSION TITLE
C++部分如何调试？

### DISCUSSION THREAD

**Discussion by freedom-shen** [openkraken/kraken#149] [Q&A] [answered]
如题，菜鸟一枚，需要对C++部分进行部分的改造，是否一定需要将C++打成对应平台的动态库进行盲调？

**Comment by wssgcg1213** [comment]
`scripts/` 目录下有配置好的打包脚本，调试方式遵循各自平台的 IDE 就好了 (比如 macOS/iOS 得用 xcode)

**Comment by andycall** [comment, marked as answer]
https://github.com/openkraken/website/blob/main/docs/guide/contribute/development.md
